prophyc/tests/generators/test_python.py

```diff
@@ -27,13 +27,26 @@ def test_includes_rendering():
     ]
 
     ref = """\
-from foo import number_12, symbol_1
-from ni_knights import definition, pi, symbol_2
-from numbers import (
-    number_0, number_1, number_10, number_11, number_13, number_14, number_15,
-    number_16, number_17, number_18, number_19, number_2, number_3, number_4,
-    number_5, number_6, number_7, number_8, number_9
-)
+if sys.version_info < (3,):
```

Comment:
I'd probably prefer to have one block of all imports for py2 and one for py3, instead of having if-else for each import statement.

Reply:
It's not that simple, it would require quite a rework of  `TranslatorBase` because right now it calls `translate_include` on every `include` node separately.